consensus/babe/babe.go

```diff
@@ -148,6 +152,7 @@ func (b *Session) invokeBlockAuthoring() {
 			} else {
 				hash := block.Header.Hash()
 				log.Info("BABE", "built block", hash.String(), "number", block.Header.Number)
+				log.Debug("BABE built block", "header", block.Header, "body", block.Body)
```

Comment:
Seems a bit excessive to have a log info and log debug stacked like this... Should we consolidate into one?

Reply:
I don't need to see the full block if I'm at info, but I wanted to see it at debug.. not sure how to consolidate

Reply:
That makes sense. It's fine as is, just noticed this has not been done before.